site/en/guides/border-router/external-commissioning/cli.md

```diff
@@ -72,7 +117,49 @@ device.
 
 ## Join the Thread network
 
-Next, on the Joiner device, [join the Thread network](join.md) and test network connectivity.
+Next, on the Joiner device, [join the Thread network](join.md) and test network
+connectivity.
+
+## Troubleshooting
+
+If you're having issues with OT Commissioner, make sure you use `stop`, then
+`exit` to properly end a session:
+
+```
+> stop
+[done]
+> exit
+```
```

Comment:
This seems to be a bug with OT Commissioner, the `exit` command should already do whatever `stop` will do. Would you create an issue with more details?

Reply:
I added this from what I read here: https://github.com/openthread/ot-commissioner/blob/main/src/app/cli/README.md#stop
I plan on updating all of the CLI Command readme's very soon...I'll make sure to revise...

Reply:
Thanks, yes I agree that the CLI readme should make it clear that the command `exit` will do first `stop`.